roiextractors/segmentationextractor.py

```diff
@@ -16,7 +16,10 @@ class SegmentationExtractor(ABC, BaseExtractor):
 
     def __init__(self):
         BaseExtractor.__init__(self)
-        self._sampling_frequency = None
+        self._sampling_frequency = np.float('NaN')
+        self._channel_names = ['OpticalChannel']
+        self._raw_movie_file_location = ''
```

Comment:
Why does `SegmentationExtractor` need a path to the raw movie file location? That should be handled by `ImagingExtractor`.

Reply:
yes, correct, 
I think this would be redundant when we are able to link the functionality of imagingextractor and segmentationextractor object: meaning if we can run the segmentation pipeline (eg. SIMA, suite2p) from an imagingextractor object and store the output. Like in spikeinterface they have run methods of spikesorters classes that take in a recording object, run the pipeline and output a sorting object; in this case both the sorting and recording object pairs would point to the same raw file, no?

Reply:
Only the recording extractor should point to the raw data file